scrap/api.py

```
@@ -0,0 +1,51 @@
+    total = math.inf
+    items = []
+
+    headers = {
+        "X-Naver-Client-Id": CLIENT_ID,
+        "X-Naver-Client-Secret": CLIENT_SECRET,
+    }
+
+    try:
+        while cur < min(total, MAX_START):
+            params = {"query": query, "display": PAGE_SIZE, "start": cur}
+            response = get(url=URL, headers=headers, params=params)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)